explorer/src/lib/components/list-item/ListItem.css

```diff
@@ -0,0 +1,64 @@
+.details-list__term {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 0.625rem;
+  padding: 0.9375rem 1.125rem;
+  background-color: var(--background-color-alt);
+  text-transform: uppercase;
+}
+
+.details-list__term:first-child {
+  border-top-left-radius: var(--control-border-radius-size);
+  border-top-right-radius: var(--control-border-radius-size);
+  padding: 1rem 1.125rem 0.9375rem;
+}
+
+.details-list__term:last-of-type {
+  border-bottom-left-radius: var(--control-border-radius-size);
+  border-bottom-right-radius: var(--control-border-radius-size);
+  padding: 0.9375rem 1.125rem 1rem;
+}
+
+.details-list__definition {
+  display: flex;
+  flex-direction: row;
+  gap: 0.625rem;
+  align-items: center;
+}
+
+@media (max-width: 640px) {
```

Comment:
Again the point for mobile-first.

I am fine with whatever approach we decide to use, but I think we have been following mobile-first until now.